### DISCUSSION TITLE
Fare checker after booking another flight

### DISCUSSION THREAD

**Discussion by jwin3** [jdholtz/auto-southwest-check-in#99] [General]
So I have the script running in docker. I was notified of a price decrease and went in to rebook to get the credit, do I need to rerun the script again for it to look for an even lower price or will it know the flight was rebooked. 

Second question is related. I have docker currently running. I booked a new flight, will it automatically detect that new flight and look for lower fares if I'm using account login?

Thanks for this awesome code. It has worked perfectly for me so far and I appreciate your hard work.

**Comment by jdholtz** [comment]
> do I need to rerun the script again for it to look for an even lower price or will it know the flight was rebooked.

This depends on if the confirmation number changed or not. If it didn’t, the script should pick up the new flight price and look for an even lower fare.

If the number did change and you have it I monitoring your account, it should automatically pick up the new flight and start monitoring that for lower fares (I haven’t added functionality to remove the old flight yet, so you may just want to restart it anyways). If you just have it monitoring the flight, you will need to restart the script and add the new confirmation number. 

> I booked a new flight, will it automatically detect that new flight and look for lower fares if I'm using account login?

Yes.
